docs/source/bundles.rst

```diff
@@ -0,0 +1,275 @@
+Data Bundles
+------------
+
+A data bundle is a collection of pricing data, adjustment data, and an asset
+database. Bundles allow us to preload all of the data we will need to run
+backtests and store the data for future runs.
+
+Ingesting Data
+~~~~~~~~~~~~~~
+
+The first step to using a data bundle is to ingest the data. This will invoke
+some custom bundle command and then write the data to a standard location that
+zipline can find. By default this location is ``$ZIPLINE_ROOT/data/<bundle>``
+where by default ``ZIPLINE_ROOT=~/.zipline``. This step may take some time as it
+could involve downloading and processing a lot of data. This can be run with:
+
+.. code-block:: bash
+
+   $ python -m zipline ingest <bundle>
+
+
+where ``<bundle>`` is the name of the bundle to ingest.
+
+Old Data
+~~~~~~~~
+
+When the ``ingest`` command is used it will write the new data to a subdirectory
+of ``$ZIPLINE_ROOT/data/<bundle>`` which is named with the current date. This
+makes it possible to look at older data or even run backtests with this older
+copy. This makers it easier to reproduce backtest results later.
+
+One drawback of saving all of this data by default is that the data directory
+may grow quite large even if you do not want to use the data. To solve this
+problem there is another command ``clean`` which will clear data bundles based
+on some time constraints.
+
+For example:
+
+.. code-block:: bash
+
+   # clean everything older than <date>
+   $ python -m zipline clean <bundle> --before <date>
+
+   # clean everything newer than <date>
+   $ python -m zipline clean <bundle> --after <date>
+
+   # keep everything in the range of [before, after] and delete the rest
+   $ python -m zipline clean <bundle> --before <date> --after <after>
```

Comment:
Will it keep everything in the range `(,after] ∪ [before,)` if I specify a `before` that follows the `after`?